src/components/PeriodDimension/PeriodTransfer.js

```diff
@@ -23,13 +25,20 @@ const PeriodTransfer = ({
     rightFooter,
     excludedPeriodTypes,
 }) => {
+    const { systemInfo = {} } = useConfig()
```

Comment:
Not related to the calendar implementation itself, but do we need to have defaults for these hooks?
We never used it this way, doesn't the hook return a default already to make sure it works without the consumer having to worry about it?

Reply:
It's the app adaptor that defines the provider:
https://github.com/dhis2/app-platform/blob/ac0350a2562429b941833725b711d1b01c0539ad/adapter/src/components/ServerVersionProvider.js#L57-L72

Reply:
I don't see it updated?